media-source.js

```diff
@@ -171,7 +171,6 @@
     'media-data-processing-steps-list': { func: videoref_helper, fragment: 'media-data-processing-steps-list', link_text: 'media data processing steps list', },
     'delaying-the-load-event-flag': {func: videoref_helper, fragment: 'delaying-the-load-event-flag', link_text: 'delaying-the-load-event-flag', },
     'intrinsic-width-and-height': { func: videoref_helper, fragment: 'concept-video-intrinsic-width', link_text: 'intrinsic width and height',  },
-    'normalized-timeranges-object': { func: videoref_helper, fragment: 'normalised-timeranges-object', link_text: 'normalized TimeRanges object',  },
```

Comment:
Is that intended? You didn't replace the occurrences of `<a def-id="normalized-timeranges-object"></a>` with `[=normalized TimeRanges object=]`, so that definition is still needed.

Reply:
oh oops. I'll revert this and do it as a followup.